recipe/meta.yaml

```diff
@@ -34,6 +39,7 @@ build:
     - "$RPATH/ld-linux-x86-64.so.2"
     - "$RPATH/libm.so.6"
   skip: true  # [osx]
+  rpaths_patcher: lief  # [ppc64le]
```

Comment:
```suggestion
  rpaths_patcher: LIEF  # [ppc64le]
```

Reply:
Thanks! Yeah, I couldn't find docs on this feature

Reply:
Though lowercase seemed to work too